SIL.Windows.Forms.Scripture/VerseControl.cs

```diff
@@ -817,8 +828,12 @@ private void HandlePasteScriptureRef()
 			// chapter and verse is optional in regex, make it 1 if not given
 			if (string.IsNullOrEmpty(chapter))
 				chapter ="1";
+			else if (chapter.Length > 3 && Regex.IsMatch(chapter, @"^\d+$"))
```

Comment:
I suspect that the actual resulting behavior here would not be what a user would expect. I think maybe if the chapter or verse portion is longer than 3 digits, we would just want to treat it like any other invalid pasted reference. It's most likely a typo or something and any one of the extra digits could be the errant one.